cli/types/index.d.ts

```diff
@@ -2934,6 +2942,130 @@ declare namespace Cypress {
     (fn: (currentSubject: Subject) => void): Chainable<Subject>
   }
 
+  /**
+   * These events come from the application currently under test (your application). These are the most useful events for you to listen to.
+   * @see https://docs.cypress.io/api/events/catalog-of-events.html#App-Events

```

Comment:
One thing we REALLY do not want is to put direct urls to any resource, because these are likely to change. The same url should go through our `on.cypress.io` redirect service. Usually it is just document title, this url would be `https://on.cypress.io/catalog-of-events#App-Events`

Reply:
Oh. Thanks. That was one of my review comments. I was trying to figure out what the redirect URL was

Reply:
Updated the URLs. Is there a conventions for those URL redirects? I tried a few including `https://on.cypress.io/events/catalog-of-events#App-Events`